components/automate-ui/src/app/pages/+compliance/shared/reporting/stats.service.ts

```diff
@@ -258,7 +258,7 @@ export class StatsService {
     return filters;
   }
 
-  formatFilters(reportQuery: ReportQuery) {
+  formatFilters(reportQuery: ReportQuery, requestsLast24h = true) {
     const apiFilters = reportQuery.filters.reduce((formatted, filter) => {
         let type = filter['type']['name'];
         let value = filter['value']['id'] || filter['value']['text'];
```

Comment:
Consider dot notation to make it less noisy here.

Reply:
I'll save this suggestion for a separate cleanup commit since this will increase the diff on this function quite a bit.